fiftyone/utils/torch.py

```diff
@@ -485,6 +503,43 @@ def __call__(self, pil_image_or_tensor):
         size = (int(round(alpha * h)), int(round(alpha * w)))
         return F.resize(pil_image_or_tensor, size, **self._kwargs)
 
+class MaxResize(object):
```

Comment:
nit: our style uses two newlines between top-level functions/classes. Can you add one more here please?

Reply:
I've just fixed the issues you addressed. Feels good to give back something since I really like this library. Good work!